voluptuous/humanize.py

```diff
@@ -0,0 +1,39 @@
+from voluptuous import Invalid, MultipleInvalid
+from voluptuous.error import Error
+
+
+MAX_VALIDATION_ERROR_ITEM_LENGTH = 500
+
+
+def _nested_getitem(data, path):
+    for item_name in path:
+        try:
```

Comment:
Would you prefer this or the functional `reduce` method mentioned above? Actually, it occurs to me that missing-value handling might not be required here, and if it is, we probably need to handle IndexError as well.